Assets/PatchKit Patcher/Scripts/UI/Languages/PatcherLanguages.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using PatchKit.Unity.Patcher.Debug;
+using PatchKit.Unity.Utilities;
+using UnityEngine;
+
+namespace PatchKit.Unity.UI.Languages
+{
+    public class PatcherLanguages : MonoBehaviour
```

Comment:
This class can be made completely static (no need to inherit from `MonoBehaviour`).

Reply:
done